cloudsensei/lambda_function.py

```diff
@@ -77,7 +77,8 @@ def get_resources(self):
                             long_running_instances_by_user.setdefault(user.lower(), {}).setdefault(region_name, []).append(
                                 {'InstanceId': resource.get('InstanceId'),
                                  'Name': name, 'LaunchDate': str(launch_time),
-                                 'RunningDays': f"{days} days", 'State': resource.get('State', {}).get('Name')})
+                                 'RunningDays': f"{days} days", 'State': resource.get('State', {}).get('Name'),
+                                 'InstanceType': resource.get('InstanceType')})
```

Comment:
can you change the order also here 'InstanceType' before 'RunningDays' ?

Reply:
No need, It is the data storing/ returning variable.